app/assets/stylesheets/_devise_error_messages.scss

```diff
@@ -0,0 +1,3 @@
+.message_width {
+  max-width: 500px;
```

Comment:
Fixed width will not scale will user's screen resolution. If you have to make it shorter, maybe you could use proportion instead.